docker-bake.hcl

```diff
@@ -24,7 +24,14 @@ group "default" {
 		"vmclarity-cr-discovery-server",
 		"vmclarity-orchestrator",
 		"vmclarity-ui",
-		"vmclarity-ui-backend"
+		"vmclarity-ui-backend",
+    		"scanner-plugin-kics",
```

Comment:
should we make this target name more consistent with the rest? something like `vmclarity-plugin-kics`